Services.Test/Storage/DocumentDb/DocumentDbRecordTest.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Exceptions;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Storage.DocumentDb;
+using Xunit;
+
+namespace Services.Test.Storage.DocumentDb
+{
+    public class DocumentDbRecordTest
```

Comment:
add a test for the ctor: `ItIsNotLockedAndNotExpiredByDefault`

Reply:
Done.